distribution/lib/Standard/Base/0.0.0-dev/src/IO.enso

```diff
@@ -29,8 +30,8 @@ print_err message = @Builtin_Method "IO.print_err"
      Print the message "Oh yes!" to standard output.
 
          IO.println "Oh yes!"
-println : Any -> Nothing
-println message = @Builtin_Method "IO.println"
+println : Any -> Text -> Nothing
+println message nl='\n' = IO_Helpers.println message nl
```

Comment:
Please let's not use such shorthands in our public (1) API. I'd suggest to call this argument `newline` or we could do similar to Python: `end` or perhaps a more explicit `end_with = '\n'` (I think I like `end_with` the most, but that's just subjective).

Also the documentation should be updated to include the added argument in the `Arguments:` section.

> (1) The `PRIVATE` comment at the top may be confusing. It is used there to prevent a text-oriented method from showing up in the GUI's component browser. But the `println` method is definitely part of our 'official' API and so it should be well documented for end-users. As [already mentioned](https://github.com/enso-org/enso/issues/9762#issuecomment-2178791785) we probably should create a separate tag, like `TEXT_ONLY` or `HIDDEN`, for documenting components that are **not internal** but should not show up in the GUI Component Browser.